src/Entities/EventMessage.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Softonic\TransactionalEventPublisher\Entities;
+
+class EventMessage
+{
+    public $service;
+
+    public $eventType;
+
+    public $modelName;
+
+    public $eventName;
+
+    public $createdAt;
+
+    public $payload;
+
+    public $meta;
+
+    /**
+     * @return string
+     */
+    public function __toString()
+    {
+        return serialize($this->toArray());
+    }
+
+    public function toJson()
```

Comment:
It is better to use the JsonSerialize standard.

Reply:
Good point. In the next commit will be implemented